### DISCUSSION TITLE
The geometric domain problem of fpde appears as otImplementedError: Rectangle mindist2boundary to be implemented.

### DISCUSSION THREAD

**Discussion by wangxiao2019** [lululxvi/deepxde#1464] [Q&A]
Hello!
When I was calculating the fractional order pde in two dimensions of space and one dimension of time, I set it to
geom = dde.geometry.Rectangle([0, 0], [1, 1])
timedomain = dde.geometry.TimeDomain(0, 1)
Geomtime = dde. Geometry. GeometryXTime (the java.awt.geom timedomain), data = dde. Data. TimeFPDE ()
Why does integer dde.data.TimePDE() work but fractional dde.data.TimeFPDE() get an error
The error is as follows:

NotImplementedError: Rectangle mindist2boundary to be implemented.
How can I solve the above situation?

**Comment by lululxvi** [comment]
It is not implemented yet.

**Comment by wangxiao2019** [comment]
您好！老师，请问一下pdf中的这种情况该如何解决呀


拼命三郎
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "lululxvi/deepxde"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月21日(星期四) 凌晨4:56
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [lululxvi/deepxde] The geometric domain problem of fpde appears as otImplementedError: Rectangle mindist2boundary to be implemented. (Discussion #1464)





 
It is not implemented yet.
